src/overlays/actors/ovl_Obj_Demo/z_obj_demo.c

```diff
@@ -29,14 +29,90 @@ const ActorInit Obj_Demo_InitVars = {
     (ActorFunc)NULL,
 };
 
-#endif
+void ObjDemo_Init(Actor* thisx, GlobalContext* globalCtx) {
+    ObjDemo* this = THIS;
+    s16 temp_v0;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Demo/ObjDemo_Init.s")
+    thisx->params &= 0xFF;
```

Comment:
This ought to be done with a GET params macro, and the special case 0xFF should have a define, at least until we find out what this is.

Reply:
two macros? please specify for each

Reply:
```suggestion
    thisx->params = OBJDEMO_GET_FF(thisx);
```
Also a `#define OBJDEMO_FF_FF 0xFF` for the comparision one line below.